app/agents/voice/automatic/features/llm_wrapper.py

```
@@ -122,6 +171,91 @@ async def wrapped_function(params):
+        
+        try:
+            return await self._original_get_chat_completions(params)
+        except BadRequestError as e:
+            # Check if this is an Azure content filter error
+            # The error message contains the error details as a string
+            error_str = str(e)
+            
+            # Check for content filter violation
+            if "content_filter" in error_str and "ResponsibleAIPolicyViolation" in error_str:
+                logger.warning(
+                    f"Azure content filter triggered (jailbreak detection). Returning safe fallback message."
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)